src/ui/components/App.tsx

```diff
@@ -103,7 +129,7 @@ function App({ theme, recordingId, modal, updateNarrowMode }: AppProps) {
 
   useEffect(() => {
     setUserInBrowserPrefs(auth.user);
-    if (auth.user) {
+    if (auth.user && !userData?.internal) {
```

Comment:
This will probably not work as intended: when you use the `useQuery` hook in a component, that component is usually rendered twice (the first time while the query is still loading and the second time with the loaded result data, unless the query has already been cached): during the first render, `userData` is `null` and hence `!userData?.internal` is `true`, so you will create a Logrocket session before finding out if this is an internal user or not. Use something like `if (auth.user && userData && !userData.internal)` instead.

Reply:
Ah good call @hbenl, I was unclear on how `?.` would behave in this situation.